AttributeError: module 'prjxray.util' has no attribute 'gen_sites'

Tried to run 
```
make[2]: Entering directory '/big/github/SymbiFlow/prjxray/fuzzers/014-ffsrcemux'
rm -rf specimen_[0-9][0-9][0-9]/ seg_clblx.segbits vivado*.log vivado_*.str vivado*.jou design *.bits *.dcp *.bit top.v run.ok
make[2]: Leaving directory '/big/github/SymbiFlow/prjxray/fuzzers/014-ffsrcemux'
make database
make[2]: Entering directory '/big/github/SymbiFlow/prjxray/fuzzers/014-ffsrcemux'
bash generate.sh specimen_001
+ source /big/github/SymbiFlow/prjxray/utils/genheader.sh
++ '[' -z artix7 ']'
++ set -ex
++ test 1 = 1
++ test '!' -e specimen_001
++ mkdir specimen_001
++ cd specimen_001
+ python3 ../top.py
Traceback (most recent call last):
  File "../top.py", line 74, in <module>
    loc = next(slices)
  File "../top.py", line 12, in gen_slices
    for _tile_name, site_name, _site_type in util.gen_sites(['SLICEL',
AttributeError: module 'prjxray.util' has no attribute 'gen_sites'
Makefile:15: recipe for target 'specimen_001/OK' failed
make[2]: *** [specimen_001/OK] Error 1
make[2]: Leaving directory '/big/github/SymbiFlow/prjxray/fuzzers/014-ffsrcemux'
Makefile:19: recipe for target 'run' failed
make[1]: *** [run] Error 2
make[1]: Leaving directory '/big/github/SymbiFlow/prjxray/fuzzers/014-ffsrcemux'
Makefile:15: recipe for target '014-ffsrcemux/run.ok' failed
make: *** [014-ffsrcemux/run.ok] Error 2
```

--- a/fuzzers/014-ffsrcemux/top.py
+++ b/fuzzers/014-ffsrcemux/top.py
@@ -9,8 +9,8 @@ print('//Requested CLBs: %s' % str(CLBN))
 
 
 def gen_slices():
-    for _tile_name, site_name, _site_type in util.gen_sites(['SLICEL',
-                                                             'SLICEM']):
+    for _tile_name, site_name, _site_type in util.get_roi().gen_sites(
+        ['SLICEL', 'SLICEM']):
         yield site_name
 
 

